ui/src/components/GeneralStatic.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
```

Comment:
Nit: for the sake of consistent style, please use single quotes here, and for `./Footer`  on the next line.